wiki/installing-and-updating-drivers.md

```diff
@@ -0,0 +1,205 @@
+
+1. On the search engine of your choice (Google, Bing, etc…), search for "{the OEM's name} drivers".
+ 
+2. Search for the model number of the piece of hardware you want to upgrade/install drivers for.
+ 
+3. When you select the product, you may have two options:
+ 
+    - Download the driver downloader (recommended).
+    - Download this specific driver only and install manually.
+
+See [Reinstalling GPU drivers from the OEM's website](/wiki/reinstalling-gpu-drivers.html#method-3-oem-website) for more information on installing GPU drivers.
+## Troubleshooting 🔧
+### If you can access Windows:
+1. Open the Device Manager by pressing <kbd>Win</kbd> + <kbd>X</kbd> and then clicking the "Device Manager" option. You can also search for it in the Start Menu.
+
+<img src="./img/installing-and-updating-drivers/troubleshooting/device-manager-power-user-menu.png" alt="The Win+X 'power user' menu, with Device Manager highlighted." width=200px>
+
+2. In Device Manager, find the device that is causing problems (you might have to expand a category), right-click on it, and click "Properties".
+
+<img src="./img/installing-and-updating-drivers/troubleshooting/device-manager-properties-button.png" alt="The context menu for a display adapter in Device Manager, with Properties highlighted." width=500px>
+
+3. Go to the "Driver" tab at the top of the window, and then click "Roll Back Driver".
+
+<img src="./img/installing-and-updating-drivers/troubleshooting/device-manager-properties-roll-back-driver.png" alt="The properties window for a device in Device Manager, with the Driver tab and Roll Back Driver button highlighted." width=500px>
+
+4. Windows will ask you why you're rolling back to a previous driver. Select a reason, and click "Yes". If you want to, you can leave a detailed response in the "Tell Us More" field, at the bottom of the window.
+
+<img src="./img/installing-and-updating-drivers/troubleshooting/driver-rollback-confirmation.png" alt="The confirmation dialogue presented after selecting Roll Back Driver." width=500px>
+
+5. Windows will then restore your driver to the previous version, which could take up to 5-10 minutes.
+
+<img src="./img/installing-and-updating-drivers/troubleshooting/hardware-change-restart-prompt.png" alt="The prompt to restart your computer, shown after rolling back a driver." width=500px>
+
+### Safe Mode
+Safe mode is a diagnostic tool, which loads Windows with only the essential drivers. This allows you to access Windows even if the problematic driver is causing issues during normal boot.
+1. Enter Safe Mode:
+
+
+On Windows 10 or 11:
+
+- Open the Start menu, click on the power button, and hold <kbd>Shift</kbd> while clicking Restart. This will take you to the Windows Recovery Environment:
+
+<img src="./img/installing-and-updating-drivers/winre/winre.png" alt="A screenshot of the Windows Recovery Environment." height=350px>
+
+- Click:
+ - Troubleshoot
```

Comment:
image formatting here is kinda messed up. re-work needed

Reply:
What's wrong with the image?

Reply:
They are... not in the right place... to say the least.

Reply:
With commits of itsWindows11 and some change I'm going to commit, solve this issue.